tests/dummy/app/controllers/application.js

```diff
@@ -1,40 +1,40 @@
 import Controller from '@ember/controller';
-import EmberObject from '@ember/object';
+import { action } from '@ember/object';
 import Changeset from 'ember-changeset';
```

Comment:
Just to check, can you check the function based approach?

```js
import { Changeset } from 'ember-changeset';
```

Reply:
I'm currently in vacation. Can't check before end of next week. 😟

Reply:
Done in d59192dc3f7756b75edfa42d9d8c10749620d698. Same results.

Reply:
Should be an easy change over for ya.  The class based approach will be deprecated at some point.